MoEngagePluginBase/MoEngagePlugin.swift

```diff
@@ -27,6 +25,16 @@ import MoEngageInApps
         initializeMoEngageDefaultInstance(sdkConfig: sdkConfig, launchOptions: launchOptions)
     }
     
+    private func initializeMoEngageDefaultInstance(sdkConfig: MOSDKConfig, sdkState: MoEngageSDKState) {
+     #if DEBUG
+            MoEngage.sharedInstance().initializeDefaultTestInstance(with: sdkConfig, andSDKState: sdkState)
+     #else
+            MoEngage.sharedInstance().initializeDefaultLiveInstance(with: sdkConfig, andSDKState: sdkState)
+    #endif
+        
+    commonSetUp(identifier: sdkConfig.identifier)
```

Comment:
@RakshithaAcharya Check for indentation here

Reply:
done